src/type/types.cpp

```diff
@@ -1865,7 +1858,7 @@ std::string ProtocolTypeToString(ProtocolType type) {
   return "INVALID";
 }
 
-ProtocolType StringToProtocolType(const std::string &str) {
+ProtocolType StringToProtocolType(const std::string& str) {
```

Comment:
Revert the remainder of these types of changes to make the pointers/references right-associative.

Reply:
When I ran clang-format on my local machine it automatically formatted the code this way... Weird, but I'll figure it out.